Cargo.toml

```diff
@@ -8,7 +8,7 @@ authors = [
 ]
 build = "build.rs"
 exclude = ["Gir*.toml", "tests/**/*", "*.md", "generator.py"]
-edition = "2021"
+edition = "2018"
```

Comment:
Please don't introduce edition regressions.

Reply:
How can i resolve this issue?